test/gui/shared/scripts/helpers/SyncHelper.py

```diff
@@ -0,0 +1,80 @@
+import re
+
+
+# File syncing in client has the following status
+SYNC_STATUS = {
+    'SYNC': 'STATUS:SYNC',  # sync in progress
+    'OK': 'STATUS:OK',  # sync completed
+    'ERROR': 'STATUS:ERROR',  # sync error
+    'IGNORE': 'STATUS:IGNORE',  # sync ignored
+    'NOP': 'STATUS:NOP',  # not in sync yet
+    'REGISTER': 'REGISTER_PATH',
+    'UNREGISTER': 'UNREGISTER_PATH',
+    'UPDATE': 'UPDATE_VIEW',
```

Comment:
I think comment in this status will be nice

Reply:
Yeah, but I don't know exactly what they mean.